cfsdk/src/main/java/io/harness/cfsdk/logging/CfLogging.java

```diff
@@ -0,0 +1,32 @@
+package io.harness.cfsdk.logging;
+
+public interface CfLogging {
+
+    void v(final String tag, final String message);
+
+    void d(final String tag, final String message);
+
+    void i(final String tag, final String message);
+
+    void w(final String tag, final String message);
+
+    void e(final String tag, final String message);
+
+    void v(final String tag, final String message, final Throwable throwable);
+
```

Comment:
I assume these func names stand for verbose, debug, info, warn, error etc ?   I'm not sure if there is a existing  convention with android, but it may be worth using the full names rather than a shortened version for readability/maintainability ?

Reply:
@davejohnston We can switch to full names. However, these are by Android convention - consistent with Android Log class method names.

Reply:
@milos85vasic-harness in that case, then this seems reasonable to follow the android convention.